proto/src/determined/run/v1/run.proto

```diff
@@ -109,4 +109,6 @@ message FlatRun {
   bool parent_archived = 18;
   // Data related the the experiment associated with this run.
   optional FlatRunExperiment experiment = 19;
+  // Number of days to retain logs for.
+  optional int32 log_retention_days = 20;
```

Comment:
is this set or returned anywhere?

Reply:
actually, I don't think so. i'll remove it